amundsen_application/static/js/components/common/BadgeList/index.spec.tsx

```diff
@@ -38,10 +38,11 @@ describe('BadgeList', () => {
       expect(container.exists()).toBe(true);
     });
 
-    it('renders a <Flag> for each badge in the input', () => {
-      expect(badgeList.find(Flag).length).toEqual(badges.length);
+    it('renders a <ClickableBadge> for each badge in the input', () => {
+      expect(badgeList.find(ClickableBadge).length).toEqual(badges.length);
     });
 
+    // TODO refactor to be clickable badge isntead of flag
```

Comment:
What is intended by this todo?

Reply:
The test right below it refers to Flag but the component is now Clickable badge so I need to update the test to reflect this